error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/BigDecimalRulesTestInput.java

```diff
@@ -17,7 +17,11 @@ ImmutableSet<BigDecimal> testBigDecimalTen() {
     return ImmutableSet.of(BigDecimal.valueOf(10), BigDecimal.valueOf(10L), new BigDecimal("10"));
   }
 
-  ImmutableSet<BigDecimal> testBigDecimalFactoryMethod() {
-    return ImmutableSet.of(new BigDecimal(0), new BigDecimal(0L));
+  ImmutableSet<BigDecimal> testBigDecimalLongFactoryMethod() {
+    return ImmutableSet.of(new BigDecimal(2), new BigDecimal(2L));
```

Comment:
felt `2` was a nicer value to use here as there is a rule to rewrite 0 to `ZERO` as well